webpack.config.js

```diff
@@ -279,9 +214,12 @@ const buildPlugin = ( env ) => {
 	const buildDir = path.resolve( __dirname, 'build' );
 	const to = path.resolve( buildDir, env.plugin );
 	const from = path.resolve( __dirname, 'plugins', env.plugin );
-	const dependencies = pluginsWithBuild.includes( env.plugin )
-		? [ `${ env.plugin }` ]
-		: [];
+	// Ensures minification and the plugin's build process (if defined) run before building the plugin.
+	const dependencies = [ 'minify-plugin-assets' ];
+
+	if ( pluginsWithBuild.includes( env.plugin ) ) {
+		dependencies.push( `${ env.plugin }` );
```

Comment:
While this was already present, this can be simplified:

```suggestion
		dependencies.push( env.plugin );
```